packages/components/tags/tag.component.ts

```diff
@@ -295,7 +304,7 @@ export class KbqTag
     /** The value of the tag. Defaults to the content inside `<kbq-tag>` tags. */
     @Input()
     get value(): any {
-        return this._value !== undefined ? this._value : this.elementRef.nativeElement.textContent;
+        return isUndefined(this._value) ? this.elementRef.nativeElement.textContent?.trim() : this._value;
```

Comment:
Можно через nullish coalescing

```suggestion
        return this._value ?? this.elementRef.nativeElement.textContent?.trim();
```